network/src/test/java/com/aria/network/ExampleUnitTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.aria.network
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```

Comment:
Could we remove this test?

Reply:
Sure, also I'm removing the androidTest. Thought about remove the test dependencies too, because from now there are no test for this module and the library of test is not used.